src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -12,5 +12,63 @@
             <tag name="kernel.event_subscriber" />
             <tag name="monolog.logger" channel="console" />
         </service>
+
+        <service id="command.about" class="Symfony\Bundle\FrameworkBundle\Command\AboutCommand">
```

Comment:
dunno.. it seems to be `<extension>.command.name`, except for framework :) there's no `framework.` prefix. Hence namespaced under root.